ansiblelint/rules/RestrictCommandsCheck.py

```diff
@@ -0,0 +1,14 @@
+from ansiblelint import AnsibleLintRule
+
+class RestrictCommandsCheck(AnsibleLintRule):
+  id = '702'
+  shortdesc = 'Warn against usage of bash utilities like awk/sed/cut/grep'
+  description = 'Certain commands like awk/sed/cut/grep should not be use'
```

Comment:
...It is recommended to use ansible/python native commands against bash utils like awk/sed/cut/grep

Reply:
done